pom.xml

```diff
@@ -13,7 +13,7 @@
   <inceptionYear>2004</inceptionYear>
 
   <prerequisites>
-    <maven>2.2.1</maven>
+    <maven>3.2.2</maven>
```

Comment:
We should probably remove the whole `<prerequisites>` section as that is only relevant when building maven plugins https://issues.apache.org/jira/browse/MNG-6092